tools/project_manager/project_quick_check.php

```diff
@@ -14,31 +14,70 @@
 
 // get data passed into the page
 $projectid = validate_projectID("projectid", @$_REQUEST['projectid'], true);
+$callback = urldecode(array_get($_REQUEST, "callback", ""));
+
+$result = [];
+
+if($callback && $projectid)
+{
+    $project = new Project($projectid);
+    if($project->user_can_do_quick_check())
+    {
+        $result = get_test_results($projectid);
+        // loop thorugh results looking for errors
+        $found_error = FALSE;
+        foreach($result as $function => $values)
+        {
+            if($result[$function]["status"] == _("Error"))
+            {
+                $found_error = TRUE;
+                break;
+            }
+        }
+
+        if(!$found_error)
+        {
+            metarefresh(0, $callback);
+        }
+    }
+}
 
 $title = _("Project Quick Check");
-$page_text = _("This page tests the project in an attempt to uncover some common errors.");
 
 output_header($title, NO_STATSBAR);
 
 echo "<h1>$title</h1>";
 
-echo "<p>$page_text</p>";
-
-// show the form
-echo "<form method='GET'>";
-echo "<table>";
-echo  "<tr>";
-echo   "<td>" . _("Project ID") . "</td>";
-echo   "<td><input name='projectid' type='text' value='$projectid' size='40' required></td>";
-echo  "</tr>";
-echo "</table>";
-echo "<input type='submit' value='Test'>";
-echo "</form>";
+if($callback)
+{
+    echo "<p>";
+    echo _("One or more errors were found when checking the project. Review the project quick check results, address any problems found, and try the project transition again.");
```

Comment:
The only thing that will trigger displaying the PQC results at this stage is an error in loading the text. How about:
"One or more errors were found when loading the text files into the project. Please review the errors listed in the Bad Bytes in Page Text (add link) section, below.

Reply:
There are a few checks that return Errors, this MR just elevates the bad bytes check to that level. I don't want to focus the language on any one check.